live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -62,3 +62,8 @@
     - { regexp: '^RPCMOUNTDOPTS=', line: 'RPCMOUNTDOPTS="--num-threads=5 --manage-gids"' }
 
 - command: systemctl disable bind9.service isc-dhcp-server.service isc-dhcp-server6.service
+
+#
+# delphix-platform installs ntp in a disabled state by default.
```

Comment:
IMO, a better comment would explain why we want to have NTP enabled.. e.g. perhaps comment on the concerns Basil raised in https://github.com/delphix/delphix-platform/pull/323#pullrequestreview-726837663

Reply:
Good point, I've updated the comment.